boa/test/__init__.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+"""
+Test utils borrowed from Brownie.
+"""
+
+import warnings
+
+from hypothesis.errors import HypothesisDeprecationWarning
+
+from .strategies import strategy  # NOQA: F401
+
+# hypothesis warns against combining function-scoped fixtures with @given
+# but in brownie this is a documented and useful behaviour. In boa.test we
```

Comment:
why is this useful?

Reply:
i removed this -- i think we should go with hypothesis on this one instead of brownie. function scoped fixtures are indeed an antipattern in hypothesis since the scoping is to the test item instead of each fuzzer case, so it doesn't do what it looks like it does. (note that as of this PR, we do get fuzz case isolation because of the HypothesisHandle monkey patch)

Reply:
cf. https://github.com/HypothesisWorks/hypothesis/issues/377#issuecomment-593091383